core/src/test/java/jenkins/model/RssFeedItemTest.java

```diff
@@ -0,0 +1,50 @@
+package jenkins.model;
+
+import hudson.model.Run;
+import hudson.scm.ChangeLogSet;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
```

Comment:
I would strongly discourage use of Mockito (or Powermock, or anything like it) when testing Jenkins code.

Reply:
Ok, I will rework the test then.